tox.ini

```diff
@@ -13,6 +13,18 @@ deps =
 commands = 
     py.test {posargs}
 
+[testenv:py36-noopenssl]
+pip_pre = False
+deps =
+    requests{env:REQUESTS_VERSION:>=2.0.1,<3.0.0}
+    pytest
+    mock
+    ndg-httpsclient
+    betamax>0.5.0
+commands =
+    pip uninstall -y pyopenssl
```

Comment:
This shouldn't be installed into this tox venv if not specified in deps above, should probably be removed

Reply:
This is *un*installing it. I already omitted it from the dependencies but it was getting installed anyway. It seems to be a dependency from one of the other test dependencies.